wcomponents-theme-parent/pom.xml

```diff
@@ -125,7 +128,7 @@
 							<goal>unpack</goal>
 						</goals>
 						<configuration>
-							<skip>${theme.skip.antrun}</skip>
+							<skip>${theme.skip.unpack}</skip>
```

Comment:
@ricksbrown Changed the unpack of i18n to be used by projects that build their own theme. So instead of "skipping" on "antrun" it now skips on "unpack".